docs/src/developer_folder/contributing.rst

```diff
@@ -29,10 +29,10 @@ Finally it is advised to create a dedicated virtual environment for this and ins
 - ``mamba create -n dev_env``
 - ``mamba activate dev_env``
 - ``cd`` to the location of the folder where you downloaded or cloned the repository.
-- install the package as a developer with test tools using the command ``pip install -e ".[dev]"``.
+- install the package as a developer with test tools using the command either by using the provided script ``python install-packages.py -d``.
+    - it calls `pip install -e` on each pymodaq package (pymodaq_utils, pymodaq_data, pymodaq_gui and pymodaq) in the package folder.
     - the `-e` option tells pip to install the package in editable mode, meaning that any change done to the source will be reported to the installed package.
-    - the `".[dev]"` argument indicate to install the current folder as a package alongside its optional dependancies, declared as `dev` in the `pyproject.toml` file.
-    - the quotes around `[dev]` just allow to escape the sequence in order to not be interpreted by the shell.
+    - the `-d` option activates `[dev]` in the package names transmitted to `pip install` (e.g. `pymodaq_utils[dev]`). It indicate to install the package alongside its optional dependancies, declared as `dev` in the `pyproject.toml` file.
```

Comment:
then can we do -e -d with your script? . Maybe you should be more explicit on what it means the [dev] optional install